### ISSUE TITLE
native support for arm64 architecture for M1 chip?

### ISSUE THREAD

**Issue by anthrotype** [justvanrossum/fontgoggles#247]
I was trying to follow the instructions in the README.md to build from source using my M1 mac. I installed the "universal2" build of Python3.10 from python.org which includes both x86_64 and arm64 architectures, set up the virtual environment and installed the requirements. I wanted to do an editable install so I run the shell script in `Turbo/build_lib.sh` like the README says, but then I saw the following warning:

```
$ ./Turbo/build_lib.sh
+ cd ./Turbo
+ mkdir -p build
+ cc -g -fPIC -c -mmacosx-version-min=10.9 -o build/makePathFromOutline.o makePathFromOutline.m
+ cc -dynamiclib -mmacosx-version-min=10.9 -o ../Lib/fontgoggles/mac/libmakePathFromOutline.dylib -framework AppKit -arch x86_64 -lsystem.b build/makePathFromOutline.o
ld: warning: ignoring file build/makePathFromOutline.o, building for macOS-x86_64 but attempting to link with file built for unknown-arm64
```

If I add `-arch arm64` flag (in addition to the `-arch x86_64` one), then it builds without warning and the dylib contains both architectures, not just intel.

```diff
diff --git a/Turbo/build_lib.sh b/Turbo/build_lib.sh
index fdbe7a6..db5cad8 100755
--- a/Turbo/build_lib.sh
+++ b/Turbo/build_lib.sh
@@ -7,6 +7,6 @@ cd "${0%/*}"  # cd into the folder containing this script

 mkdir -p build

-cc -g -fPIC -c -mmacosx-version-min=10.9 -o build/makePathFromOutline.o makePathFromOutline.m
+cc -g -fPIC -c -mmacosx-version-min=10.9 -arch arm64 -arch x86_64 -o build/makePathFromOutline.o makePathFromOutline.m

-cc -dynamiclib -mmacosx-version-min=10.9 -o ../Lib/fontgoggles/mac/libmakePathFromOutline.dylib -framework AppKit -arch x86_64 -lsystem.b build/makePathFromOutline.o
+cc -dynamiclib -mmacosx-version-min=10.9 -o ../Lib/fontgoggles/mac/libmakePathFromOutline.dylib -framework AppKit -arch arm64 -arch x86_64 -lsystem.b build/makePathFromOutline.o
```

Then I was able to run the test suite and do a `python App/setup.py py2app -A` to create an alias app and run it.

**Comment by justvanrossum** [comment]
Did you see #244?

**Comment by justvanrossum** [comment]
Should be fixed by #251
